server_admin/topics/users/proc-setting-default-required-actions.adoc

```diff
@@ -0,0 +1,16 @@
+[id="proc-setting-default-required-actions_{context}"]
+= Default Required Actions
+
+You can also specify required actions that will be added to an account in two ways.  
+* When a new user is created, use the *Add User* button on the user list screen
+* Use the <<_user-registration, user registration>> link on the login page.  
+
+
+You can specify the default required actions 
+
+.Procedure
+. Click *Authentication* in the left menu.
+. Click the `Required Actions` tab.
+. Click the checkbox in the `Default Action` column of the required actions that you want to be executed when a brand new user logs in.
```

Comment:
How about "Click the checkbox in the `Default Action` column for one or more required actions. When a new user first logs in, those actions will be executed."